posthog/temporal/workflows/backfill_batch_export.py

```diff
@@ -77,7 +87,11 @@ async def get_schedule_frequency(schedule_id: str) -> float:
     )
 
     handle = client.get_schedule_handle(schedule_id)
-    desc = await handle.describe()
+
+    try:
+        desc = await handle.describe()
+    except temporalio.service.RPCError:
```

Comment:
Same question as above, `RPCError` feels so broad that this is a little scary to me. But it could be my own inexperience with Temporal?

Reply:
This one could be scarier. But I think we can double check the status code, like you pointed out.

Reply:
I've added a commit to check the status of the `RPCError`: if it's `NOT_FOUND` then we assume schedule not found (and raise our own `TemporalScheduleNotFoundError`). Otherwise, we re-raise the exception.